src/main/java/uk/gov/hmcts/darts/transcriptions/mapper/TranscriptionResponseMapper.java

```diff
@@ -58,6 +59,15 @@ public static TranscriptionResponse mapToTranscriptionResponse(TranscriptionEnti
             transcriptionResponse.setCourthouse(courtCase.getCourthouse().getCourthouseName());
             transcriptionResponse.setDefendants(courtCase.getDefendantStringList());
             transcriptionResponse.setJudges(courtCase.getJudgeStringList());
+            transcriptionResponse.setFrom(transcriptionEntity.getCreatedBy().getUsername());
+
+            if (transcriptionEntity.getTranscriptionStatus() != null) {
+                transcriptionResponse.setStatus(transcriptionEntity.getTranscriptionStatus().getStatusType());
+            }
+
+            transcriptionResponse.setReceived(transcriptionEntity.getCreatedDateTime());
+            transcriptionResponse.setComments(transcriptionEntity.getTranscriptionCommentEntities().stream().map(
```

Comment:
I don't think it's as simple as this unfortunately.  We need to check the transcription_workflow where we should have a transcription_status: 1 | Requested 

This will be the "Comments to the transcriber (Optional)" i.e. Requestor Instructions.
I believe although the schema supports Many - this will actually be zero or one for the instructions.

`transcription_workflow where tra_id = <transcription_id> and trs_id = 1`
`-> transcription_comment where trw_id = ?`
The entities probably need a tweak to facilitate the navigation, but once you have identified the right workflow entry, then you should be able to use workflow_actor | workflow_ts for the other 2 bits of info.

Reply:
I have addressed this as much as I can. I have not yet made the workflow dependency a zero or one.

Reply:
I am currently returning the comments from both the workflow relationship as well as the one to many between the transaction and transaction_comment table.